extras/linked-token/script/CallLockAndTransfer.s.sol

```diff
@@ -0,0 +1,18 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.23;
+
+import "forge-std/Script.sol";
+import "../src/LinkedTokenController.sol";
+
+contract CallLockAndTransfer is Script {
```

Comment:
Can we turn this into a cast command in the Makefile? (was review comment before already)

Reply:
pushed  817e6e75